examples/v1beta1/trial-images/pytorch-mnist/requirements.txt

```diff
@@ -1,4 +1,4 @@
 cloudml-hypertune==0.1.0.dev6
-torch==1.13.1
-torchvision==0.14.1
+torch==2.2.1
+torchvision==0.17.1
```

Comment:
Should we also update the PyTorch base image in our SDK: https://github.com/kubeflow/katib/blob/master/sdk/python/v1beta1/kubeflow/katib/constants/constants.py#L55 ?

Reply:
Sure.

Reply:
Done.